frontend/src/login/SignUpBox.tsx

```diff
@@ -1,5 +1,23 @@
-import { Image, Button, Card, CardBody, Heading, Stack, FormControl, FormLabel, Input } from '@chakra-ui/react'
+import { useState } from 'react';
+import { Image, Button, Card, CardBody, Heading, Stack, FormControl, FormLabel, Input, CardFooter, Text, Link } from '@chakra-ui/react'
+import SignUpHandler from './SignUpHandler';
+import UsernameInput from '../components/UsernameInput'
+import PasswordInput from '../components/PasswordInput'
+
 const SignUpBox = () => {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const [repeatPassword, setRepeatPassword] = useState('');
+
+  const handleSubmit = () => {
+    if (password !== repeatPassword) {
+      alert('Passwords do not match');
```

Comment:
Ikke noget der behøver at blive lavet:
Jeg synes det kunne være federe hvis vi får elementer til at vise at der er fejl istedet for at arbejde med alerts. De bliver utroligt træge at arbejde med på sigt